plugins/kotlin-as-java/src/main/kotlin/converters/KotlinCompanion.kt

```diff
@@ -0,0 +1,78 @@
+package org.jetbrains.dokka.kotlinAsJava.converters
+
+import org.jetbrains.dokka.links.Callable
+import org.jetbrains.dokka.model.*
+import org.jetbrains.dokka.model.properties.PropertyContainer
+
+private const val DEFAULT_COMPANION_NAME = "Companion"
+
+internal class KotlinCompanion(private val companion: DObject?) {
```

Comment:
Looks like there is no need in this class as it doesn't have any state as is. Instead, we could move members to top-level and make them extension to `DObject`

Reply:
I thought about it but didn't want to add confusion with additional methods to usual named objects